src/lib/app/versions.tsx

```diff
@@ -34,13 +33,13 @@ export async function normalizeToAppVersionUuid(
       `${await getAppNameFromUuid(
         apiClient,
         appUuid,
-      )} version ${version} does not seem to exist for the mStudio.`,
+      )} Version ${version} does not seem to exist for the mStudio.`,
```

Comment:
Capitalization was correct before.

```suggestion
      )} version ${version} does not seem to exist for the mStudio.`,
```